src/Cli/Utils.cs

```diff
@@ -904,27 +904,37 @@ public static bool WriteJsonContentToFile(string file, string jsonContent)
         /// This method will check if DAB_ENVIRONMENT value is set.
         /// If yes, it will try to merge dab-config.json with dab-config.{DAB_ENVIRONMENT}.json
         /// </summary>
-        public static string MergeConfigsIfAvailable()
+        /// <returns>Returns the name of the merged Config if successful.</returns>
+        public static bool TryMergeConfigsIfAvailable(out string mergedFile)
         {
             string? environmentValue = Environment.GetEnvironmentVariable(RUNTIME_ENVIRONMENT_VAR_NAME);
+            mergedFile = string.Empty;
             if (!string.IsNullOrEmpty(environmentValue))
             {
-                string FileToBeOverridden = DefaultName;
-                string FileToBeOverriddenWith = $"{CONFIGFILE_NAME}.{environmentValue}{CONFIG_EXTENSION}";
-                string mergedFile = $"{CONFIGFILE_NAME}.{environmentValue}.merged{CONFIG_EXTENSION}";
+                string baseConfigFile = DefaultName;
```

Comment:
nit: DefaultName -> DefaultFileName or DefaultConfigFileName

Reply:
`RuntimeConfigPath.DefaultName` which translates to default config file name, i.e., `dab-config.json`